### ISSUE TITLE
New reference on Tupi-Guarani family tupi1276

### ISSUE THREAD

**Issue by thiagochacon** [glottolog/glottolog#1076]
Important reference for the Tupi-Guarani migration hypothesis

@article{souza_mello_novas_2017,
	title = {Novas evidências linguísticas (e algumas arqueológicas) que apontam para a origem dos povos tupi-guarani no leste amazônico},
	issn = {0716-5811},
	url = {http://www.scielo.cl/scielo.php?script=sci_arttext&pid=S0716-58112017000200299&lng=en&nrm=iso&tlng=en},
	doi = {10.4067/S0716-58112017000200299},
	abstract = {The most common procedure to search for evidence about the geographical location of a proto-people is called “Linguistic Migration Theory”. The area that concentrates the greatest linguistic diversity within the family is the most likely to be the origin of the proto-population. The dialogue with another sciences (archeology, etnohistory and comparative culture) is fundamental for a better composition of the past. There are four major hypothesis for the Tupi-Guarani family origin and dispersion, three based on linguistic data and one based on archaeological evidence. The linguistic hypothesis supposes that the origin of the TupiGuarani family in a similar place of the Tupi stock. We shall demonstrate here that the area of highest Tupi-Guarani family diversity of the family does is not the same of the Tupi stock. This diversity can be proved with pholological parameters and lexical isoglosses that delimitate some linguistic subgroups.},
	language = {pt},
	number = {36},
	urldate = {2024-05-09},
	journal = {Literatura y lingüística},
	author = {Souza Mello, Antônio Augusto and Kneip, Andreas},
	month = dec,
	year = {2017},
	pages = {299--312},
	file = {Souza Mello and Kneip - 2017 - Novas evidências linguísticas (e algumas arqueológ.pdf:/Users/thiagochacon/Zotero/storage/8W9ZCVB2/Souza Mello and Kneip - 2017 - Novas evidências linguísticas (e algumas arqueológ.pdf:application/pdf},
}

[Souza Mello and Kneip - 2017 - Novas evidências linguísticas (e algumas arqueológ.pdf](https://github.com/glottolog/glottolog/files/15267130/Souza.Mello.and.Kneip.-.2017.-.Novas.evidencias.linguisticas.e.algumas.arqueolog.pdf)

**Comment by d97hah** [comment]
Thanks a lot, will add for next version!

Pada Kam, 9 Mei 2024 pukul 22.05 Thiago Chacon ***@***.***>
menulis:

**Comment by xrotwang** [comment]
To be included in Glottolog 5.1:
```
$ grep "10.4067/S0716-58112017000200299" references/bibtex/*
references/bibtex/hh.bib:    doi = {10.4067/S0716-58112017000200299},
```
